### ISSUE TITLE
`dmtcp_get_local_status()` return values are not in sync with its documentation

### ISSUE THREAD

**Issue by rohgarg** [dmtcp/dmtcp#224]
This looks like a documentation bug. The `dmtcp_get_local_status()` function returns 1 on success and 3 (`DMTCP_NOT_PRESENT`) on failure, but the documentation suggests that it should return 0 on success and -1 on failure.

**Comment by gc00** [comment]
@rohgarg, Thanks for your quick response.  Could we make two more addition to your patch?  It's ugly to return `1` or `DMTCP_NOT_PRESENT`.  It should always return a macro.  (It's my fault.  I wrote the original code.)

Let's add after `#ifndef DMTCP_NOT_PRESENT` the following code.
```C
#ifndef DMTCP_IS_PRESENT
# define DMTCP_IS_PRESENT 4
#endif
```

Then, let's change the documentation of `dmtcp_get_local_status` to:
```C
 * + Returns DMTCP_IS_PRESENT on success and DMTCP_NOT_PRESENT on error.
```
and after that, let's change the logic to:
```C
  (dmtcp_get_local_status ? (dmtcp_get_local_status(c,r), DMTCP_IS_PRESENT) : \
                            DMTCP_NOT_PRESENT)
```

**Comment by rohgarg** [comment]
Closed via PR #225
